docs/index.bs

```diff
@@ -3132,6 +3158,17 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Return |response|.
   </section>
 
+  <section algorithm>
+    <h3 id="skippable-fetch-handler-algorithm"><dfn>Skippable Fetch Handler</dfn></h3>
+      : Input
+      :: |serviceWorker|, a [=/service worker=]
+      : Output
+      :: a boolean
+
+      1. If |serviceworker| has |has non-no-op fetch listeners flag|, and the value is false, then the user agent *may* return true.
```

Comment:
What's the reason for "may" here, given that "No-Op Handler identification" already uses "may"?

Reply:
I just tried to follow what "Should Skip Event" does, but it does not need to be *may*.  Let me omit.